src/providers/DataProviders.test.js

```diff
@@ -31,7 +31,9 @@ const commonData = {
   instanceStatuses: [{ id: 'id-1', code: 'batch' }],
   instanceFormats: [{ id: 'id-1', code: 'sb' }],
   holdingsSources: [{ id: 'id-1', name: 'FOLIO' }],
-  instanceTypes: [{ id: 'id-1', code: 'crd' }]
+  instanceTypes: [{ id: 'id-1', code: 'crd' }],
+  subjectSources: [{ id: 'id-1', name: 'source' }],
+  subjectTypes: [{ id: 'id-1', name: 'type' }],
```

Comment:
The `DataProvider` takes data from two places the `resources` and `useCommonData`. The last one is used in both `ui-inventory` and `ui-plugin-find-instance`. The `subjectSources` and `subjectTypes` are part of `resources`, so they shouldn't be in the `commonData` variable.

Reply:
fixed, thanks